component/src/main/wso2/kafka/producer.bal

```diff
@@ -47,15 +47,15 @@ import ballerina/system;
 # + transactionTimeout - Timeout for transaction status update from the producer.
 # + enableIdempotence - Exactly one copy of each message is written in the stream when enabled.
 public type ProducerConfig record {
-    string? bootstrapServers; // BOOTSTRAP_SERVERS_CONFIG 0
-    string? acks; // ACKS_CONFIG 1
-    string? compressionType; // COMPRESSION_TYPE_CONFIG 2
-    string? clientID; // CLIENT_ID_CONFIG 3
-    string? metricsRecordingLevel; // METRICS_RECORDING_LEVEL_CONFIG 4
-    string? metricReporterClasses; // METRIC_REPORTER_CLASSES_CONFIG 5
-    string? partitionerClass; // PARTITIONER_CLASS_CONFIG 6
-    string? interceptorClasses; // INTERCEPTOR_CLASSES_CONFIG 7
-    string? transactionalID; // TRANSACTIONAL_ID_CONFIG 8
+    string? bootstrapServers = (); // BOOTSTRAP_SERVERS_CONFIG 0
```

Comment:
Is it essential to provide a value for these configs ? can't we stick with  i.e string bootstrapServers? instead. Same might be applicable for other params as well

Reply:
Sounds good, but this needs some changes in the parsing the arguments in the server implementation. Shall we address this later ?